amd_openvx/openvx/ago/ago_platform.h

```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
 
 // OpenCL: enabled unless disabled explicitly by setting ENABLE_OPENCL=0
 #ifndef ENABLE_OPENCL
-#define ENABLE_OPENCL  1
```

Comment:
This needs be turned on by default. Only use -D option in CMake to turn off while building.